paper.tex

```diff
@@ -1185,10 +1185,76 @@ \section*{ARG simplification}
+backwards in time approach, it has led to substantial
+insights into the process of recombination itself
+and to the sequentially Markov Coalescent (SMC)
+approximations~\citep{mcvean2005approximating,marjoram2006fast}
+which are central to many modern methods.
+
+Under the coalescent with recombination (or SMC) each local
+tree along the genome is transformed into the next by
+a ``subtree prune-and-regraft'' (SPR)
+% Hein et al call it the "subtree transfer" operation but it's
+% the same thing.
+operation~\citep{hein1990reconstructing,song2003on,song2006properties}.
+The number of SPRs required to transform one leaf-labelled tree
+into another is known as the SPR distance
+% Hein 1996 say that SPR distance is NP-hard, but Allen 2001
+% say this is wrong. But then Bordewich went ahead and said
+% yes, actually it is NP hard.
+and is NP-hard to
+compute~\citep{hein1996complexity,allen2001subtree,bordewich2005computational}.
+
+[Discussion of SPRs in an EARG]
+
+The effects of recombination as we move from left-to-right along
+the genome in an unsimplified GARG are easy to interpret.
+Consider the leftmost tree in Fig~\ref{fig-simplification}.
+In the first tree the parent of B is F, and in the second
+tree its parent is E. Since both parents are present as
+unary nodes in the tree, we simply change one edge in
+order to effect the transition.
+A more complex transition occurs when we have a regraft operation
+that occurs above the current root (i.e., the current oldest
+non-unary node). For example, the transition between the
+second and third trees corresponds to node D switching
+from the path through I to H, and we then must add branches
+corresponding to passing through L and meeting at M.
+
+The equivalence between an ARG and a set of local trees
+separated by SPRs is often mentioned [e.g. x y z], but
+it is important to note that while extracting local
+trees from an ARG is straightforward, the converse
+process of reconstructing precisely the same ARG from
+these trees is more problematic. If the internal
+nodes are consistently labelled and unary nodes are
+included in the local trees (e.g., Fig~\ref{fig-simplification})
+then it is easy to exactly reconstruct the ARG.
+Otherwise, the problem is much more difficult and it
+may not be possible to uniquely reconstruct the original ARG.
+Without labels for the internal nodes, we are forced to
+to reason about which subtrees remain the same and which
+differ due to the SPR separating adjacent trees in order to
+assign node identity. Without the internal unary nodes marking
+recombinations, we must infer their existence. In
+general, there will not be a unique solution to this problem,
+and so there can be many possible ARGs compatible with
+a given list of
+% want to say "standard phylogenetic" trees here or something, but
+% and they don't have to be binary (just arity > 1), but thought
+% this would be familiar to readers?
+leaf-labelled binary trees.
```

Comment:
Could just say "marginal trees", "ancestral trees", "local trees", or something like that. Then there is no implication of binarity, but readers should still be on familiar ground.

Reply:
I want to contrast with things that contain unary nodes though, which are quite unfamiliar to people I think (indeed, it seems like you can't have them in R's phylo objects)

I've added your note to the text anyway, something to mull for next pass.